src/main/java/mat/server/service/impl/FhirMeasureServiceImpl.java

```diff
@@ -28,7 +38,7 @@
 
 @Service
 public class FhirMeasureServiceImpl implements FhirMeasureService {
-
+    public static boolean TEST_MODE = false;
```

Comment:
could you please make it private?

Reply:
no i have to set this in test cases to get around hard coded hibernet lob stuff in our daos.

Reply:
How about package private, then?

Reply:
package private then codacy will complain :(

Reply:
Right, forgot about that rule. I'll have to verify, but I'm pretty sure using package private here is a best practice from [Effective Java](https://learning.oreilly.com/library/view/effective-java-3rd/9780134686097/).